### ISSUE TITLE
about transformer framework

### ISSUE THREAD

**Issue by chenweihua91** [Tencent/PocketFlow#20]
Is there a plan to support the transformer framework?

**Comment by jiaxiang-wu** [comment]
Yes, we have plans to support RNN/LSTM-based models, in the next few months.

**Comment by jiaxiang-wu** [comment]
Enhancement required: add support for RNN/LSTM-based models, *e.g.* Transformer.

**Comment by GaofengCheng** [comment]
@jiaxiang-wu any progress in supporting LSTM/GRU? I'm from ASR filed and very interested in seeing the effect of pocketflow in LSTM/GRU.
